src/constants/component-specific.ts

```diff
@@ -60,3 +60,11 @@ export type PopupPlacement = (typeof POPUP_PLACEMENTS)[number];
 
 /** Gauge */
 export const GAUGE_RADIUS = 100;
+
+/** Breadcrumb */
+
+export interface BreadcrumbPath {
```

Comment:
why did it end up in constants? If you want to make it global, normally we add types in `.d.ts` files. Or if it is only needed in one place, then in that file. We also probably need to export it.

Reply:
let's not talk about it ahah 😄